dockers/docker-fpm-frr/TSA

```diff
@@ -1,11 +1,18 @@
 #!/bin/bash
 
+# Check whether the routemap is for internal BGP sessions.
+function is_internal_route_map()
```

Comment:
can we make this common function instead of defining in each of TSA/B/C

Reply:
@abdosi updated, I have added more common functions which was repeated in all files.